api/v1alpha3/machineset_webhook.go

```diff
@@ -35,9 +36,16 @@ func (m *MachineSet) SetupWebhookWithManager(mgr ctrl.Manager) error {
 }
 
 // +kubebuilder:webhook:verbs=create;update,path=/validate-cluster-x-k8s-io-v1alpha3-machineset,mutating=false,failurePolicy=fail,groups=cluster.x-k8s.io,resources=machinesets,versions=v1alpha3,name=validation.machineset.cluster.x-k8s.io
+// +kubebuilder:webhook:verbs=create;update,path=/mutate-cluster-x-k8s-io-v1alpha3-machineset,mutating=true,failurePolicy=fail,groups=cluster.x-k8s.io,resources=machinesets,versions=v1alpha3,name=default.machineset.cluster.x-k8s.io
 
+var _ webhook.Defaulter = &MachineSet{}
 var _ webhook.Validator = &MachineSet{}
 
+// DefaultingFunction sets default MachineSet field values.
+func (m *MachineSet) Default() {
+	PopulateDefaultsMachineSet(m)
```

Comment:
Is `PopulateDefaultsMachineSet` used anywhere else? Can we just merge the two functions?

Reply:
it's not. I can merge the two!